app/config/config.yml

```diff
@@ -34,6 +35,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    globals:
+        google_map_api_key: AIzaSyBITmQHpnrjkl8CQOjd8x9JqgATOUESxEU
```

Comment:
To be moved to the parameters.yml (and I suggest you rebase to not keep this info in the history)